pkg/scheduler/framework/runtime/framework.go

```diff
@@ -594,6 +595,101 @@ func (f *frameworkImpl) runPostFilterPlugin(ctx context.Context, pl framework.Po
 	return r, s
 }
 
+// RunFilterPluginsWithNominatedPods runs the set of configured filter plugins
+// for nominated pod on the given node.
+func (f *frameworkImpl) RunFilterPluginsWithNominatedPods(ctx context.Context, state *framework.CycleState, pod *v1.Pod, info *framework.NodeInfo) (bool, *framework.Status, error) {
```

Comment:
we don't need to return the bool, caller can check if the pod fits the node using `status.IsSuccess()`.

Reply:
Yes, the original bool here may be caused by the wired status behavior. I have created an issue #97267 to track it.